charts/cluster-api-provider-aws/crds/awsmanagedclusters.infrastructure.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+    service.beta.openshift.io/inject-cabundle: "true"
+  labels:
+    cluster.x-k8s.io/provider: infrastructure-aws
+    cluster.x-k8s.io/v1alpha3: v1alpha3
+    cluster.x-k8s.io/v1alpha4: v1alpha4
+    cluster.x-k8s.io/v1beta1: v1beta1_v1beta2
+  name: awsmanagedclusters.infrastructure.cluster.x-k8s.io
+spec:
+  conversion:
```

Comment:
This must be removed , It failed the installation. check [here](https://github.com/serngawy/cluster-api-installer/blob/capahelm/src/cluster-api-provider-aws/kustomization.yaml#L26) example how to remove it using kustomize  after removing the caBundle field.